src/Components/fe-header/FeHeader.js

```diff
@@ -3,44 +3,41 @@ import { LitElement, html, css } from 'lit-element';
 export class FeHeader extends LitElement {
   static get properties() {
     return {
-      title: { type: String },
-      btnLabel: { type: String },
+      title: { type: String }
+     
     };
   }
 
   static get styles() {
     return css`
       :host {
-        min-height: 100vh;
+        height:15vh;
         display: flex;
         flex-direction: column;
         align-items: center;
         justify-content: flex-start;
-        font-size: calc(10px + 2vmin);
-        color: #1a2b42;
-        max-width: 960px;
+        font-size: 25px;
+        color: #fff;
+        
+        background-color: var(--fe-header-background-color);
```

Comment:
My suggestion is keep background color static, its not needed to change it from other components.
only text change is fine c